custom/enikshay/case_utils.py

```diff
@@ -169,44 +172,52 @@ def get_all_episode_cases_from_person(domain, person_case_id):
     return episode_cases
 
 
-def get_open_episode_case_from_occurrence(domain, occurrence_case_id):
+def get_open_active_dstb_episode_case_from_occurrence(domain, occurrence_case_id):
     """
-    Gets the first open 'episode' case for the occurrence
+    In eNikshay public sector app
+    Gets the first open DSTB 'episode' case for the occurrence
 
     Assumes the following case structure:
     Occurrence <--ext-- Episode
 
     """
     case_accessor = CaseAccessors(domain)
     episode_cases = case_accessor.get_reverse_indexed_cases([occurrence_case_id])
-    open_episode_cases = [case for case in episode_cases
-                          if not case.closed and case.type == CASE_TYPE_EPISODE and
-                          case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
-    if open_episode_cases:
-        return open_episode_cases[0]
+    open_active_episode_cases = [case for case in episode_cases
+                                 if not case.closed and
+                                 case.type == CASE_TYPE_EPISODE and
+                                 case.get_case_property().get('episode_type') == "confirmed_tb" and
```

Comment:
I don't think this syntax works. You need either `get_case_property('episode_type')` or `dynamic_case_properties().get('episode_type')`

Reply:
oh no.. intended `get_case_property('episode_type')`.